### PR TITLE
Upgraded to dotnet 1.1

### PR DESCRIPTION
**Pull request by jimmyp**
@humblelistener @mattbirman @hombredequeso @ceolaflannery @ColinScott

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ before_install:
   - sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list'
   - sudo apt-key adv --keyserver apt-mo.trafficmanager.net --recv-keys 417A0893
   - sudo apt-get update
-  - sudo apt-get install -y dotnet-dev-1.0.0-preview1-002702
+  - sudo apt-get install -y dotnet-dev-1.1.0-preview1-001153
```
Comment (mattbirman) [inline .travis.yml]: Line ending issues?

Comment (jimmyp) [reply .travis.yml]: What does this even mean?

File: WebService/project.json [hunk 2]
```diff
@@ -1,30 +1,55 @@
...
+    "Microsoft.Extensions.Logging.Debug": "1.0.0",
+    "Microsoft.VisualStudio.Web.BrowserLink.Loader": "14.0.0",
+    "Microsoft.VisualStudio.Web.CodeGeneration.Tools": {
+      "version": "1.0.0-preview2-update1",
+      "type": "build"
+    },
+    "Microsoft.VisualStudio.Web.CodeGenerators.Mvc": {
+      "version": "1.0.0-preview2-update1",
+      "type": "build"
+    }
```
Comment (mattbirman) [inline WebService/project.json:41]: Do we definitely need all of these?

**Comment by jimmyp** [comment]
Build is green 💚 can I get a LGTM?

**Comment by mattbirman** [comment]
LGTM :shipit:

**Pull request merged**